`'distributionManagement.status' must not be specified.`

@MarkEWaite found in `git-client-plugin` among others a minor regression introduced acc. `git bisect` by @basil’s #310:: a nonfatal message

```
[ERROR] 'distributionManagement.status' must not be specified. @ line 166, column 13
```

I have no idea what it means (cannot find anything interesting at line 166 in either the plugin’s POM or `plugin-pom`). Maybe coming from some unnamed dependency.

I hit this breakpoint:

```
validateEffectiveModel:457, DefaultModelValidator (org.apache.maven.model.validation)
build:564, DefaultModelBuilder (org.apache.maven.model.building)
build:454, DefaultModelBuilder (org.apache.maven.model.building)
build:267, DefaultModelBuilder (org.apache.maven.model.building)
build:173, DefaultProjectBuilder (org.apache.maven.project)
build:342, DefaultProjectBuilder (org.apache.maven.project)
build:299, DefaultProjectBuilder (org.apache.maven.project)
resolvePom:56, MavenArtifact (org.jenkinsci.maven.plugins.hpi)
getResolvedType:250, MavenArtifact (org.jenkinsci.maven.plugins.hpi)
isPlugin:63, MavenArtifact (org.jenkinsci.maven.plugins.hpi)
isPluginBestEffort:73, MavenArtifact (org.jenkinsci.maven.plugins.hpi)
execute:29, ValidateHpiMojo (org.jenkinsci.maven.plugins.hpi)
executeMojo:137, DefaultBuildPluginManager (org.apache.maven.plugin)
doExecute:301, MojoExecutor (org.apache.maven.lifecycle.internal)
execute:211, MojoExecutor (org.apache.maven.lifecycle.internal)
execute:165, MojoExecutor (org.apache.maven.lifecycle.internal)
execute:157, MojoExecutor (org.apache.maven.lifecycle.internal)
buildProject:121, LifecycleModuleBuilder (org.apache.maven.lifecycle.internal)
buildProject:81, LifecycleModuleBuilder (org.apache.maven.lifecycle.internal)
build:56, SingleThreadedBuilder (org.apache.maven.lifecycle.internal.builder.singlethreaded)
execute:127, LifecycleStarter (org.apache.maven.lifecycle.internal)
doExecute:294, DefaultMaven (org.apache.maven)
doExecute:192, DefaultMaven (org.apache.maven)
execute:105, DefaultMaven (org.apache.maven)
execute:960, MavenCli (org.apache.maven.cli)
doMain:293, MavenCli (org.apache.maven.cli)
main:196, MavenCli (org.apache.maven.cli)
invoke0:-1, NativeMethodAccessorImpl (jdk.internal.reflect)
invoke:77, NativeMethodAccessorImpl (jdk.internal.reflect)
invoke:43, DelegatingMethodAccessorImpl (jdk.internal.reflect)
invoke:568, Method (java.lang.reflect)
launchEnhanced:282, Launcher (org.codehaus.plexus.classworlds.launcher)
launch:225, Launcher (org.codehaus.plexus.classworlds.launcher)
mainWithExitCode:406, Launcher (org.codehaus.plexus.classworlds.launcher)
main:347, Launcher (org.codehaus.plexus.classworlds.launcher)
```

So the "[ERROR] 'distributionManagement.status' must not be specified. @ line 166, column 13" is coming from `commons-discovery-0.4.pom` which does indeed have this near line 166:

```xml
  <distributionManagement>
    <repository>
      <id>default</id>
      <name>Default Repository</name>
      <url>file:///www/jakarta.apache.org/builds/jakarta-commons/${pom.artifactId.substring(8)}/</url>
    </repository>
    <site>
      <id>default</id>
      <name>Default Site</name>
      <url>scp://people.apache.org//www/jakarta.apache.org/commons/${pom.artifactId.substring(8)}/</url>
    </site>
    <status>converted</status>
  </distributionManagement>
```

--- a/src/main/java/org/jenkinsci/maven/plugins/hpi/MavenArtifact.java
+++ b/src/main/java/org/jenkinsci/maven/plugins/hpi/MavenArtifact.java
@@ -6,6 +6,7 @@ import org.apache.maven.artifact.factory.ArtifactFactory;
 import org.apache.maven.artifact.versioning.ArtifactVersion;
 import org.apache.maven.artifact.versioning.OverConstrainedVersionException;
 import org.apache.maven.execution.MavenSession;
+import org.apache.maven.model.building.ModelBuildingRequest;
 import org.apache.maven.plugin.logging.Log;
 import org.apache.maven.project.DefaultProjectBuildingRequest;
 import org.apache.maven.project.MavenProject;
@@ -53,6 +54,7 @@ public class MavenArtifact implements Comparable<MavenArtifact> {
         ProjectBuildingRequest buildingRequest =
                 new DefaultProjectBuildingRequest(session.getProjectBuildingRequest());
         buildingRequest.setProcessPlugins(false); // improve performance
+        buildingRequest.setValidationLevel(ModelBuildingRequest.VALIDATION_LEVEL_MINIMAL);
         return builder.build(artifact, buildingRequest).getProject();
     }
 

